omniauth-cas3.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |gem|
   gem.version       = Omniauth::Cas3::VERSION
 
   gem.add_dependency 'omniauth',                '~> 1.2'
-  gem.add_dependency 'nokogiri',                '~> 1.6.6'
+  gem.add_dependency 'nokogiri',                '~> 1.7'
```

Comment:
I'm thinking we should probably do `'~> 1.7', '>= 1.7.1' to allow all 1.x versions but ensure that the minim is 1.7.1 since it's a security version?